prow/plugins/welcome/welcome.go

```diff
@@ -32,7 +32,7 @@ import (
 
 const (
 	pluginName            = "welcome"
-	defaultWelcomeMessage = "Welcome @{{.AuthorLogin}}! It looks like this is your first PR to {{.Org}}/{{.Repo}} 🎉"
+	defaultWelcomeMessage = "Welcome @{{.AuthorLogin}}! It looks like this is your first PR to {{.Org}}/{{.Repo}} 🎉. Please refer to our [pull request process documentation](https://git.k8s.io/community/contributors/guide/pull-requests.md) to help your PR have a smooth ride to approval. You will be prompted by a bot to use commands during the review process. Do not be afraid to follow the prompts! It is okay to experiment. [Here is the bot commands documentation](https://go.k8s.io/bot-commands). You can also check if {{.Org}}/{{.Repo}} has [its own contribution guidelines]({{.Org}}/{{.Repo}}/CONTRIBUTING.md). You may want to refer to our [testing guide](https://git.k8s.io/community/contributors/devel/sig-testing/testing.md)if you run into trouble with your tests not passing. If you are having difficulty getting your pull request seen, please follow the [recommended escalation practices](https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#why-is-my-pull-request-not-getting-reviewed). We want to make sure your contribution gets all the attention it needs! Thank you, and welcome to Kubernetes."
```

Comment:
You are missing a space between the link to testing.md and "if you run into trouble".

A paragraph break or two may help people read this better. Perhaps before "You will be prompted", "You may want to refer to our testing guide", and "If you are having difficulty" (essentially 1 paragraph per help topic).  Can we use `<br/>` in the welcome message to insert them?

Reply:
These are  great things to link to, thank you @guineveresaenger for following through on this. ❤️